Week3/homework/CreditCardValidate.js

```diff
@@ -0,0 +1,46 @@
+"use strict"
+
+function validCreditCard (num) {
+    let card = num;
```

Comment:
Ah. I don't think you need to create an extra element `card` here which is nothing but containing the same value as `num` You can use `num` instead.